crates/bevy_app/src/app.rs

```diff
@@ -42,12 +41,18 @@ bevy_utils::define_label!(AppLabel);
 /// }
 /// ```
 pub struct App {
+    /// Stores and exposes operations on [entities](bevy_ecs::entity::Entity), [components](bevy_ecs::component::Component), resources,
+    /// and their associated metadata.
     pub world: World,
+    /// A [runner function](Self::set_runner) which is typically not configured manually,
+    /// but set by a Bevy integrated plugin.
```

Comment:
I think this should say what the runner function is/does, not just that it's typically not configured manually. The docs should help people understand why they might ever want to care about the thing that's being documented.

You can split it into two sentences:
> The runner function *<specifies/determines/does>* *\<thing>*.
> Typically, it is not configured manually, but set by one of Bevy's built-in plugins. *[which one?]*